import_tracker/lazy_import_errors.py

```diff
@@ -530,3 +544,9 @@ def _get_non_import_modules(cls):
                 for frame in cls._FrameGenerator()
             ),
         )
+
+
+def _is_import_time():
```

Comment:
I think this will be caught in the other PR, but this needs a docstring and type hint

Reply:
yep, will add that. This PR would need rebasing after the other one gets merged